percona-distribution-postgresql-12/README.md

```diff
@@ -18,7 +18,7 @@ Images are updated when new releases are published.
 
 Start a Percona Distribution for PostgreSQL container as follows:
 
-    docker run --name container-name -d perconalab/percona-distribution-postgresql:tag
+    docker run --name container-name -e POSTGRESQL-PASSWORD=secret -d perconalab/percona-distribution-postgresql:tag
```

Comment:
The variable uses an underscore instead of a hyphen

Reply:
Updated